Sign_Up_Demo/Controllers/ViewController.swift

```diff
@@ -28,24 +29,24 @@ class ViewController: UIViewController {
 
 
         if fName.isEmpty || fName.isValidText {
-            showAlert(message: "Invalid First Name!\nIt must contain only Alphabats (A-Z or a-z).")
+            showAlertWithMsg(message: "Invalid First Name!\nIt must contain only Alphabats (A-Z or a-z).")
```

Comment:
For constant strings Create a file and type named "Constants" and keep all constants there and use them here.